Gordon360/ApiControllers/ProfilesController.cs

```diff
@@ -338,11 +338,25 @@ public IHttpActionResult getAdvisors(string username)
 
         }
 
+        /// <summary> Gets the clifton strengths of a particular user </summary>
+        /// <param name="username"> The username for which to retrieve info </param>
+        /// <returns> Clifton strengths of the given user. </returns>
+        [HttpGet]
+        [Route("clifton/{username}")]
+        public IHttpActionResult GetCliftonStrengths(string username)
+        {
+            var id = _accountService.GetAccountByUsername(username).GordonID;
+            var strengths = _profileService.GetCliftonStrengths(int.Parse(id));
+
+            return Ok(strengths);
+
+        }
+
```

Comment:
This route allows anyone who is authenticated to access this info, so we need to be sure it doesn't contain any sensitive or private info. 

If only a specific subset of people (e.g. FacStaff, admins, etc) should access to anyone's info besides their own, then we should add a StateYourBusiness filter here with the logic to reflect that.